### DISCUSSION TITLE
Cannot get final name for export 'default' of ./node_modules/jose/dist/node/esm/runtime/random.js

### DISCUSSION THREAD

**Discussion by yaakua** [panva/jose#633] [Q&A] [answered]
When I am building an application using Next.js with next-auth@5.0-beta, it gives me the error mentioned in the title. I have searched through all the documentation and issues but did not find a corresponding solution.

**Comment by panva** [comment, marked as answer]
Hi @yaakua, This is to be raised with your bundler of choice.

**Reply by panva** [reply]
I cannot help you with random build issues coming from various builders and bundlers, please open issues in the trackers of said tools.

**Comment by Francoisbeche** [comment]
Hello, i face the same issue on 5.0.0 beta-15, cannot find any solution on google/docs

`./node_modules/@auth/core/index.js + 110 modules
Cannot get final name for export 'default' of ./node_modules/jose/dist/node/esm/runtime/random.js`

**Reply by Prajay-Chaudhary** [reply]
@Francoisbeche  Did you find the solution?

**Reply by tanapoln** [reply]
I'm able to reproduce this. Please check my repo https://github.com/tanapoln/nextjs-build-error

Still don't know how to fix it.
